src/lib.rs

```diff
@@ -208,6 +210,13 @@ impl Sub<Timespec> for Timespec {
     }
 }
 
+#[cfg(feature = "rustc-serialize")]
+impl ToJson for Timespec {
+    fn to_json(&self) -> Json {
+        Json::String(json::encode(&self).unwrap())
```

Comment:
This is going to end up creating things like

```json
{
    "my_time": "{\"sec\": 894582834, \"nsec\": 1200}"
}
```

which is probably not what you want.